Source/missiles.cpp

```diff
@@ -3594,8 +3610,7 @@ void MI_LightningArrow(int i)
 		if ((mx != Missiles[i]._miVar1 || my != Missiles[i]._miVar2) && mx > 0 && my > 0 && mx < MAXDUNX && my < MAXDUNY) {
 			if (Missiles[i]._misource != -1) {
 				if (Missiles[i]._micaster == TARGET_PLAYERS
-				    && Monsters[Missiles[i]._misource].MType->mtype >= MT_STORM
-				    && Monsters[Missiles[i]._misource].MType->mtype <= MT_MAEL) {
+				    && IsAnyOf(Monsters[Missiles[i]._misource].MType->mtype, MT_STORM, MT_RSTORM, MT_STORML, MT_MAEL)) {

```

Comment:
Enum is defined as:
```
	MT_INVALID,
	MT_RSTORM,
	MT_STORM,
	MT_STORML,
	MT_MAEL,
	MT_WINGED,
```

So we should end up with MT_STORM, MT_STORML and MT_MAEL. MT_RSTORM isn't in the range.

```suggestion
				    && IsAnyOf(Monsters[Missiles[i]._misource].MType->mtype, MT_STORM, MT_STORML, MT_MAEL)) {

```

Reply:
Where are you looking that up???

```cpp
	MT_HELLBURN,
	MT_STORM,
	MT_RSTORM,
	MT_STORML,
	MT_MAEL,
	MT_BIGFALL,
```

Reply:
Sorry my mistake.
I looked at `MonstConvTbl` in `monsterdat.cpp`.

Reply:
Good to know :)